### DISCUSSION TITLE
Articles et autres informations intéressantes 📰

### DISCUSSION THREAD

**Discussion by EmileForest** [CUPUM/ecometropole_laurentienne#67] [General]
[Article](https://www.nytimes.com/2022/09/15/magazine/extraterrestrials-technosignatures.html?searchResultPosition=5) à propos des nouvelles stratégies pour découvrir la vie extraterrestre. Un professeur de l'Université de Terre-Neuve souligne l'importance de décoloniser ces approches; il critique l'idée de ne chercher que des civilisations qui ressembleraient à la civilisation occidentale, par exemple. 

Bien intéressant !

**Comment by EmileForest** [comment]
[Article/ essai](https://emergencemagazine.org/essay/the-serviceberry/) poétique et pertinent à propos des interrelations entre les espèces et les écosystèmes. Cet article explore également l'importante de restaurer les relations entre les humains et le "vivant" de façon plus générale. Robin est très, très nice.

**Comment by EmileForest** [comment]
@MaryanneMonette @AhmedBK95 @katherineromero20 @AlexandraDesroches 

Plantes et autres végétaux riveraines : chrome-extension://efaidnbmnnnibpcajpcglclefindmkaj/http://zipsud.org/wp-content/uploads/2014/02/Liste-vegetaux.pdf

**Comment by EmileForest** [comment]
@FlaviaCozac, ceci me semble pertinent : https://macphailwoods.org/resources/native-plant-species 

+ Il me semble que l'Herbe de feu (Epilobium angustifolium) ça serait une plante qui colonniserait facilement ce type de site.

**Reply by FlaviaCozac** [reply]
Merci!

**Reply by EmileForest** [reply]
Et des Aubépine (Crataegus) !

**Reply by EmileForest** [reply]
Je dis ça, je dis rien.

**Comment by EmileForest** [comment]
@Sarahprairie https://www.embassyofthenorthsea.com/product/rights-of-nature-case-studies-from-six-continents/

**Comment by sjinko** [comment]
@marieantoinettetayrouz : 
https://lp.ca/t7HqQr?sharing=true

**Comment by EmileForest** [comment]
@AhmedBK95 @Sarahprairie https://lateraloffice.com/MAKING-CAMP-2015

**Reply by EmileForest** [reply]
@MaryanneMonette
